fem/ceed/util.hpp

```diff
@@ -64,6 +74,14 @@ void InitBasisAndRestriction(const mfem::FiniteElementSpace &fes,
                              Ceed ceed, CeedBasis *basis,
                              CeedElemRestriction *restr);
 
+void InitTensorRestriction(const FiniteElementSpace &fes,
+                           Ceed ceed, CeedElemRestriction *restr);
+
+int CeedOperatorGetActiveField(CeedOperator oper, CeedOperatorField *field);
```

Comment:
Rename to `GetOperatorActiveField`?

Reply:
As we've discussed before in other places, I think this should stay with the C-style and Ceed-style naming conventions.

Reply:
Not completely convinced, but fine.